src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -58,21 +54,24 @@ public class User implements Serializable {
     @Column
     private Gender gender;
 
-    // to see how elementCollection works:
-    /*
-    @ElementCollection
-    @CollectionTable(name = "whiteCard", joinColumns = @JoinColumn(name = "id"))
-    @Column(name = "whiteCards")
-    private List<WhiteCard> whiteCards;
-     */
-
     @OneToOne
-    private BlackCard blackCard;
+    private Game activeGame;
 
     @OneToMany
-    private List<WhiteCard> whiteCards = new ArrayList<>();
+    private List<Game> pastGames = new ArrayList<>();
+
+    @OneToOne
+    private UserCards userCards;
```

Comment:
The reason is because we can save a date with it. Doing it how it is now, we have exactly one timestamp for the usercards (n black and m white cards), which we can use to reset the game.

Reply:
But I see that it is confusing saving unrelated cards in one repo. Splitted in half, now the whitecards are on their own and decoupled from any Timestamp.